Kernel/TTY/PTYMultiplexer.cpp

```diff
@@ -35,7 +35,7 @@ UNMAP_AFTER_INIT PTYMultiplexer::~PTYMultiplexer() = default;
 
 UNMAP_AFTER_INIT void PTYMultiplexer::initialize()
 {
-    the().after_inserting();
+    MUST(the().after_inserting());
```

Comment:
Wouldn't a `release_value_but_fixme_should_propagate_errors` be more accurate?

Reply:
I don't think so, because there's no meaning for error handling if such thing happened - the kernel should always panic if that happens, and it is probably better to have the exact line where it panicked.